code42/py42
# Fix/within-the-last-silo

### Description of Change ###

Tried to query alerts using `DateObserved.within_the_last()` and it did not work, so I made it so you can't do that, and also was confused by one of the doc strings, so I adjusted that too.

### Issues Resolved ###
Working on a scratch for testing something I am making for alerts.

### Testing Procedure ###
Make sure you can't call `within_the_last` using `DateObserved`.
Make sure you can still use `within_the_last` for `EventTimestamp`.
Make sure you can still use `within_the_last` for `InsertionTimestamp`

### PR Checklist ###
Did you remember to do the below?
